nifi-frontend/src/main/frontend/apps/standard-content-viewer/src/app/pages/standard-content-viewer/feature/standard-content-viewer.component.scss

```diff
@@ -0,0 +1,26 @@
+/*!
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+.standard-content-viewer {
+    h3 {
+        margin-bottom: 0;
```

Comment:
This seems a little unnecessary. We rely on the angular typography to style this header and we should probably not be changing that. What is the issue here? I see you also added a `mr-2` under the 'Original|Formatted' buttons. I think if you remove that margin and remove this margin on the `h3` then the layout should be correct.

Reply:
Where did you see an `mr-2` in the standard content viewer UI?